class NotUniqueException(Exception):
    pass

class ExistedException(Exception):
    pass